db/seeds.rb

```diff
@@ -5,3 +5,19 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+# Criteria and assicated records
+stakeholder_names = [ 'Local communities', 'Indigenous peoples', 'Non-governmental organisation (NGO)', 'Private sector (business and industry)', 'Sub-national or local government' ]
+Stakeholder.create!(stakeholder_names.map {|name| { name: name }})
+
+cbd_objective_names = [ 'Conservation of biological diversity', 'Sustainable use', 'Fair and equitable sharing of benefits from the utilization of genetic resources' ]
+CbdObjective.create!(cbd_objective_names.map {|name| { name: name }})
+
+criteria = [
+  { boundary: true, five_year_commitment: true, progress_report: true, stakeholder_ids: Stakeholder.pluck(:id).sample(2), cbd_objective_ids: CbdObjective.pluck(:id).sample(2) },
+  { boundary: false, five_year_commitment: true, progress_report: true, stakeholder_ids: Stakeholder.pluck(:id).sample(2), cbd_objective_ids: CbdObjective.pluck(:id).sample(2) },
+  { boundary: true, five_year_commitment: false, progress_report: true, stakeholder_ids: Stakeholder.pluck(:id).sample(2), cbd_objective_ids: CbdObjective.pluck(:id).sample(2) },
+  { boundary: true, five_year_commitment: true, progress_report: false, stakeholder_ids: Stakeholder.pluck(:id).sample(2), cbd_objective_ids: CbdObjective.pluck(:id).sample(2) },
+  { boundary: true, five_year_commitment: true, progress_report: true, stakeholder_ids: [CbdObjective.last], cbd_objective_ids: [CbdObjective.last] }
```

Comment:
I got an error when seeding this time (reset/dropped the database first): `ActiveRecord::RecordNotFound: Couldn't find Stakeholder with 'id'=[nil]`

I'm guessing in line 21 it should be `stakeholder_ids: [Stakeholder.last]` rather than `stakeholder_ids: [CbdObjective.last]`?

Reply:
you're right, fixed